airbyte-cdk/python/airbyte_cdk/sources/streams/http/requests_native_auth/oauth.py

```diff
@@ -31,6 +31,7 @@ def __init__(
         expires_in_name: str = "expires_in",
         refresh_request_body: Mapping[str, Any] = None,
         grant_type: str = "refresh_token",
+        refresh_request_body_content_type: ContentType = ContentType.X_WWW_FORM_URLENCODED,
```

Comment:
default to x-www-form-urlencoded since that's what we currently use on master. It is however possible to configure it